README.md

```diff
@@ -21,25 +21,22 @@ If users do not have a cookie issued, they can request an email with a link for
 
 If a lot users are requesting new cookies, we might have to increase the read/write capacity of the dynamo table where cookie tokens are stored.
 
-There will be a limited number of users that can switch emergency access on and off. This will be required if Google Auth
-is down.
+#### Managing emergency access
 
-Users that can change the switch will have their userId and a password hash stored in DynamoDB.
+Emergency access is configured using an S3-based configuration file, which is updated via the `emergency-access` script. This replaces the previous approach for emergency access management and provides a simple command-line interface to enable or disable emergency access.
 
-`userId` is the username of a Guardian email address e.g joe.bloggs if the email address is joe.bloggs@guardian.co.uk
-`passwordHash` is generated using [bCrypt](https://github.com/t3hnar/scala-bcrypt). To generate a hash, checkout this repository and run:
-```
-sbt console
-import com.github.t3hnar.bcrypt._
-"[password-value]".boundedBcrypt
-```
-Add a new item to the Composer DynamoDB table `login.gutools-emergency-access-[STAGE]` containing the userId and password hash.
+To manage emergency access:
 
-To turn a switch on or off run:
-```
-curl -X POST 'https://[login-domain]/switches/emergency/[on|off]' -k -H 'Authorization: Basic [firstname.lastname]@guardian.co.uk:[password]'
+```bash
+# Enable emergency access for a stage (with composer aws credentials)
+./script/emergency-access enable PROD --profile composer
+
+# Disable emergency access for a stage (with composer aws credentials)
+./script/emergency-access disable PROD --profile composer
 ```
 
+You will need composer AWS credentials to use this script. If access to AWS via Janus is an issue, break-glass credentials may need to be used. 
```

Comment:
```suggestion
You will need composer AWS credentials to use this script. If access to AWS via Janus is an issue, break-glass credentials may need to be used. In this case it might be easier to set environment variables with the credentials and omit the `--profile` option when running the script.
```